lib/mix/tasks/export.buys.ex

```diff
@@ -0,0 +1,48 @@
+defmodule Mix.Tasks.Export.Buys do
+  use Mix.Task
+
+  def run(args) do
+    Mix.Task.run("app.start")
+
+    cond do
+      length(args) == 0 ->
+        Mix.shell().info("Output: Screen")
+        export()
+
+      true ->
+        Mix.shell().info("Output: File (#{args |> List.first()})")
+        args |> List.first() |> export
```

Comment:
In heroku, we won't be able to write to files but it's cool to have this option here.

Reply:
I didn't know that 😬